### ISSUE TITLE
Cant link in gd : Use of undeclared type 

### ISSUE THREAD

**Issue by idi888** [twostraws/SwiftGD#19]
Every time I try to build, I get a bunch of Use of undeclared type. I have installed gd through brew
Cant seem to find the issue

Mojave 10.14.5

![Screen Shot 2019-06-10 at 11 11 31 am](https://user-images.githubusercontent.com/8197402/59166783-84ac6780-8b70-11e9-9a9b-e7d1375012e5.png)

**Comment by idi888** [comment]
I fixed the issue by changing hte /gd/module.modulemap header line back to /usr/local/include from umbrella header

**Comment by aggsyb** [comment]
@idi888 I am getting the exact same issue.

> /gd/module.modulemap header line back to /usr/local/include from umbrella header

Can you let me know where / what the "umbrella" header is please, not having much joy!

Thanks.

**Comment by idi888** [comment]
@aggsyb SwiftGD/Sources/gd/module.modulemap

**Comment by jpimbert** [comment]
I fixed the issue in three steps :

1. Add `/usr/local/lib` in _Library Search Paths_ in the _Build Settings_ of **SwiftGD** and **App** as indicated by @twostraws in **Server Side Swift** book

1. Change _.build/checkouts/SwiftGD/Sources/gd/module.modulemap_   
```
module gd {
        umbrella header "/usr/local/include/gd.h"
        link "gd"
}
```

3. Add `/usr/local/include` RECURSIVE in _Header Search Paths_ in the _Build Settings_ of **SwiftGD**

**Comment by siemensikkema** [comment]
@jpimbert Do you know if there is a way that doesn't require performing these steps each time the Xcode project is regenerated?

**Comment by jpimbert** [comment]
@siemensikkema, yes I think there is a way but I presently don't know where to find it 😬 
Perhaps by adding a Xcode configuration file, documented here : https://help.apple.com/xcode/mac/current/#/itcaec37c2a6 ?
Perhaps by fixing it in SwiftGD configuration ?

**Comment by andrew804** [comment]
Looks like this is fixed #20 @twostraws any chance you could merge?
